### PR TITLE
Add new FAQs to index.html

### PR DESCRIPTION
**Pull request by BalajiAI**
As referenced in [this issue](https://github.com/CORD-ai/cord-website/issues/24), I've added 2 extra FAQ to index.html.
Kindly see the changes that I've made.

### CODE REVIEW

**Comment by AneeshBose** [comment]
@BalajiAI You can also add content taking stuff from here - https://www.linkedin.com/feed/update/urn:li:activity:6939230394982486016

**Review by Anindyadeep — APPROVED** [review]
LGTM. But yes, you can take the additional motivations from there and also from any other website if required.

**Comment by BalajiAI** [comment]
@AneeshBose @Anindyadeep Yeah, I can do it. But, There're already 8 FAQs are there in the website, so I think it's okay for now (since adding more doesn't looks good). We can discuss it about in our upcoming meeting.
What ars your thoughts?

**Comment by Anindyadeep** [comment]
Yeahh that's fine.

**Comment by AneeshBose** [comment]
> @AneeshBose @Anindyadeep Yeah, I can do it. But, There're already 8 FAQs are there in the website, so I think it's okay for now (since adding more doesn't looks good). We can discuss it about in our upcoming meeting. @GM-git-dotcom also mentioned that he can also help with this. What are your thoughts?

Let's keep it 8 and prioritize the most important ones.

**Comment by BalajiAI** [comment]
> > @AneeshBose @Anindyadeep Yeah, I can do it. But, There're already 8 FAQs are there in the website, so I think it's okay for now (since adding more doesn't looks good). We can discuss it about in our upcoming meeting. @GM-git-dotcom also mentioned that he can also help with this. What are your thoughts?
> 
> Let's keep it 8 and prioritize the most important ones.

Yes !

File: index.html [hunk 1]
```diff
@@ -238,24 +249,34 @@ <h2>F.A.Q</h2>
...
             </div>
           </li>
 
           <li>
-            <div data-bs-toggle="collapse" href="#faq6" class="collapsed question">What other opportunities can I get from this platform? <i class="bi bi-chevron-down icon-show"></i><i class="bi bi-chevron-up icon-close"></i></div>
-            <div id="faq6" class="collapse" data-bs-parent=".faq-list">
+            <div data-bs-toggle="collapse" href="#faq7" class="collapsed question">What other opportunities can I get from this platform? <i class="bi bi-chevron-down icon-show"></i><i class="bi bi-chevron-up icon-close"></i></div>
+            <div id="faq7" class="collapse" data-bs-parent=".faq-list">
               <p>
                 Other opportunities include, getting referrals, LoRs (in longer terms). Other perks include, getting chances to host different sessions,
```
[the remark this answers is not in the document]
Comment (AneeshBose) [reply index.html:264]: @BalajiAI This is one thing which might need changing. Please incorporate and I will merge this.

**Review by AneeshBose — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (BalajiAI) [reply index.html]: Yeah. I've updated the answer for the faq7. Kindly see the changes.

**Review by AneeshBose — APPROVED** [review]
LGTM!

**Pull request merged**